nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml

```diff
@@ -204,181 +215,40 @@
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
+            <properties>
+                <elasticsearch_docker_image>7.17.6</elasticsearch_docker_image>
+                <elasticsearch.elastic.password>s3cret</elasticsearch.elastic.password>
+            </properties>
             <build>
                 <pluginManagement>
                     <plugins>
                         <plugin>
                             <groupId>org.apache.maven.plugins</groupId>
                             <artifactId>maven-failsafe-plugin</artifactId>
+                            <configuration>
+                                <systemPropertyVariables>
+                                    <elasticsearch.endpoint>http://localhost:9200</elasticsearch.endpoint>
+                                    <elasticsearch.testcontainers.enabled>true</elasticsearch.testcontainers.enabled>
+                                    <elasticsearch.docker.image>docker.elastic.co/elasticsearch/elasticsearch:${elasticsearch_docker_image}</elasticsearch.docker.image>
+                                    <elasticsearch.elastic_user.password>${elasticsearch.elastic.password}</elasticsearch.elastic_user.password>
+                                </systemPropertyVariables>
+                            </configuration>
                         </plugin>
                     </plugins>
                 </pluginManagement>
             </build>
         </profile>
-
-        <profile>
-            <!-- use with elasticsearch-oss only (-default can be used if x-pack-ml permissions fixed) -->
-            <id>elasticsearch-6</id>
-            <activation>
-                <activeByDefault>false</activeByDefault>
-            </activation>
-            <properties>
-                <es.int.version>6.8.23</es.int.version>
-                <es.int.type.name>_doc</es.int.type.name>
-                <es.int.script.name>setup-6.script</es.int.script.name>
-                <es.int.path.conf />
-            </properties>
-        </profile>
-        <profile>
-            <!-- use with elasticsearch-oss or elasticsearch-default -->
-            <id>elasticsearch-7</id>
-            <activation>
-                <activeByDefault>false</activeByDefault>
-            </activation>
-            <properties>
-                <es.int.version>7.10.2</es.int.version>
-                <es.int.script.name>setup-7.script</es.int.script.name>
-                <es.int.type.name />
-                <es.int.path.conf />
-            </properties>
-        </profile>
         <profile>
-            <!-- use with elasticsearch-default -->
-            <id>elasticsearch-7-no-oss</id>
-            <activation>
-                <activeByDefault>false</activeByDefault>
-            </activation>
+            <id>elasticsearch6</id>
```

Comment:
Update this module's `README.md` to instruct developers how to run the `integration-tests` in there various guises (i.e. different versions of Elasticsearch)

Reply:
Done